nntrainer/models/neuralnet.cpp

```diff
@@ -395,7 +395,7 @@ sharedConstTensors NeuralNetwork::forwarding(
       **/
       model_graph.checkLoadComplete(f);
       node->forwarding(training);
-      model_graph.inActive(f);
+      // model_graph.inActive(f);
```

Comment:
Wouldn't it be better just to delete it?

Reply:
Done Thank you for Review